waiter/resources/log4j.properties

```diff
@@ -28,3 +28,13 @@ log4j.appender.RequestLogAppender.File=log/${waiter.logFilePrefix}request.log
 log4j.appender.RequestLogAppender.DatePattern='.'yyyy-MM-dd
 log4j.appender.RequestLogAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.RequestLogAppender.layout.ConversionPattern=%m%n
+
+log4j.category.Kubernetes=DEBUG, KubernetesAppender
+log4j.additivity.Kubernetes=false
+
+log4j.appender.KubernetesAppender=org.apache.log4j.DailyRollingFileAppender
```

Comment:
I like that idea. Just rename it Scheduler, right?

Reply:
I think this is very helpful for Kubernetes debugging. I also like tailing this file during demos to show how the Waiter operations we're doing turn into Kubernetes API requests. I think it's nice having it separate. Do you have strong feelings about adding another log file?

Reply:
I initially thought you only kept it for being able to debug. I'm good with having a separate file for the scheduler.

Reply:
The Kubernetes log is now a more generic Scheduler log. I also moved the logging macro to the scheduler namespace since that seemed more fitting; i.e., `(k8s-log ...)` → `(scheduler/log ...)`.